comid/example_psa_refval_test.go

```diff
@@ -20,7 +20,6 @@ func Example_psa_refval() {
 		panic(err)
 	}
 
-	// output:
```

Comment:
if you remove the `output` keyword, I am not sure these tests are still active.

Reply:
Ah, ok, good catch,I will fix this!